func/main.sh

```diff
@@ -1074,9 +1074,11 @@ is_cron_format_valid() {
 
 # Name validator
 is_name_format_valid() {
```

Comment:
it is also used by 

object) is_name_format_valid "$arg" 'object' ;;

We should update the name to different function name